deploy/docker-compose/.env

```diff
@@ -0,0 +1,65 @@
+# Container Images
+INFORMATICS_GATEWAY_IMAGE=ghcr.io/project-monai/monai-deploy-informatics-gateway:0.3.0
+INFORMATICS_GATEWAY_CONFIG_IMAGE=dwdraju/alpine-curl-jq:latest
+WORKFLOW_MANAGER_IMAGE=ghcr.io/project-monai/monai-deploy-workflow-manager:0.1.0
+TASK_MANAGER_IMAGE=ghcr.io/project-monai/monai-deploy-task-manager:0.1.0
+
+MINIO_IMAGE=minio/minio:latest
+MINIO_MC_IMAGE=minio/mc
+MONGODB_IMAGE=mongo:latest
+ORTHANC_IMAGE=osimis/orthanc
+RABBITMQ_IMAGE=rabbitmq:3-management
+
+# Static IP allocation
+SUBNET=172.29.0.0/16
+GATEWAY=172.29.0.1
+
+INFORMATICS_GATEWAY_API_IP=172.29.0.50
+INFORMATICS_GATEWAY_API_CONFIG_IP=172.29.0.51
+WORKFLOW_MANAGER_IP=172.29.0.60
+TASK_MANAGER_IP=172.29.0.70
+
+MINIO_CONFIG_IP=172.29.0.11
+MINIO_IP=172.29.0.10
+MONGODB_IP=172.29.0.30
+ORTHANC_IP=172.29.0.100
+RABBITMQ_IP=172.29.0.20
+
+# Service ports
+# Changing the port numbers may require updates to the service configuration files
+INFORMATICS_GATEWAY_SCP_PORT=104
+INFORMATICS_GATEWAY_HL7_PORT=2575
+INFORMATICS_GATEWAY_API_PORT=5000
+WORKFLOW_MANAGER_PORT=5001
+TASK_MANAGER_PORT=5002
+
+MINIO_API_PORT=9000
+MINIO_CONSOLE_PORT=9001
+MONGODB_PORT=27017
+ORTHANC_SCP_PORT=1114
+ORTHANC_UI_PORT=8089
+RABBITMQ_MGT_PORT=15672
+RABBITMQ_PORT=5672
+
+# Service hostnames
+# Changing the host names may require updates to the service configuration files
+INFORMATICS_GATEWAY_HOSTNAME=informatics-gateway
+TASK_MANAGER_HOSTNAME=task-manager
+WORKFLOW_MANAGER_HOSTNAME=workflow-manager
+
+MINIO_HOSTNAME=minio
```

Comment:
NIT: Any reason why not grouping the host names with the default IPs next to each other? for readability

Reply:
Updated